### PR TITLE
mark dynamodb tests and add snapshots

### PR DESCRIPTION
**Pull request by ackdav**
### Summary
- Marked every dynamodb test with either only_localstack or aws_validated
- Added snapshots for each aws_validated test
- Corrected a few minor things to make snapshots validate (or added a skip_validate)
- Removed test_dynamodb_batch_write_item as it tests the same as test_batch_write_items 

In a second PR I will correct several only_localstack tests to make them aws validated...

### CODE REVIEW

File: tests/integration/test_dynamodb.py [hunk 1]
```diff
@@ -1342,33 +1348,7 @@ def test_dynamodb_stream_to_lambda(
         lambda_client.delete_event_source_mapping(UUID=mapping_uuid)
 
     @pytest.mark.aws_validated
-    def test_dynamodb_batch_write_item(
```
Comment (steffyP) [inline tests/integration/test_dynamodb.py:1345]: was the test case `test_dynamodb_batch_write_item removed`?

**Review by steffyP — COMMENTED** [review]
LGTM! Thanks for validating the tests and adding snapshots 👍 😄

there is one test case that seems to be missing now `test_dynamodb_batch_write_item`- was that on purpose?

File: tests/integration/test_dynamodb.py (hunk 1 above)
Comment (ackdav) [reply tests/integration/test_dynamodb.py:1345]: Yes I removed it. It has does almost exactly the same as `test_batch_write_items()`. Both target the function `dynamodb_client.batch_write_item()`...
`test_dynamodb_batch_write_item()` tests 3 put requests
`test_batch_write_items()` tests 1 delete- and 1 putrequest
Let me know if I'm missing something :)

Comment (steffyP) [reply tests/integration/test_dynamodb.py:1345]: ah I see - seems to be very similar indeed :) 
/cc @giograno

**Review by steffyP — APPROVED** [review]

File: tests/integration/test_dynamodb.py (hunk 1 above)
Comment (giograno) [reply tests/integration/test_dynamodb.py:1345]: Fine with dropping one :)

**Review by giograno — APPROVED** [review]

**Pull request merged**